Xamarin.Forms.Core/IndicatorsView.cs

```diff
@@ -0,0 +1,285 @@
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsStyles();
+		});
+
+		public static readonly BindableProperty CountProperty = BindableProperty.Create(nameof(Count), typeof(int), typeof(IndicatorsView), default(int), propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsCount((int)oldValue, (int)newValue);
+		});
+
+		public static readonly BindableProperty MaxVisibleCountProperty = BindableProperty.Create(nameof(MaxVisibleCount), typeof(int), typeof(IndicatorsView), int.MaxValue, propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsStyles();
+		});
+
+		public static readonly BindableProperty IndicatorTemplateProperty = BindableProperty.Create(nameof(IndicatorTemplate), typeof(DataTemplate), typeof(IndicatorsView), default(DataTemplate), propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsStyles();
+		});
+
+		public static readonly BindableProperty IndicatorShapeProperty = BindableProperty.Create(nameof(IndicatorShape), typeof(IndicatorShape), typeof(IndicatorsView), default(IndicatorShape), propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsStyles();
+		});
+
+		public static readonly BindableProperty IndicatorsColorProperty = BindableProperty.Create(nameof(IndicatorsColor), typeof(Color), typeof(IndicatorsView), Color.Silver, propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsStyles();
+		});
+
+		public static readonly BindableProperty SelectedIndicatorColorProperty = BindableProperty.Create(nameof(SelectedIndicatorColor), typeof(Color), typeof(IndicatorsView), Color.Gray, propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsStyles();
+		});
+
+		public static readonly BindableProperty IndicatorsSizeProperty = BindableProperty.Create(nameof(IndicatorsSize), typeof(double), typeof(IndicatorsView), 10.0, propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetIndicatorsStyles();
+		});
+
+		internal static readonly BindableProperty ItemsSourceProperty = BindableProperty.Create(nameof(ItemsSource), typeof(IEnumerable), typeof(IndicatorsView), null, propertyChanged: (bindable, oldValue, newValue) =>
+		{
+			(bindable as IndicatorsView)?.ResetItemsSource(oldValue as IEnumerable);
+		});
+
+		static IndicatorsView()
+		{
+		}
+
+		public IndicatorsView()
+		{
+			Orientation = StackOrientation.Horizontal;
+		}
+
+		public int Position
+		{
+			get => (int)GetValue(PositionProperty);
+			set => SetValue(PositionProperty, value);
+		}
+
+		public int Count
+		{
+			get => (int)GetValue(CountProperty);
+			set => SetValue(CountProperty, value);
+		}
+
+		public int MaxVisibleCount
```

Comment:
Maybe this needs to be renamed too. We have `MinimumHeightRequest`. Not sure if we should stick to the non-abbreviated name here and use `MaximumVisibleCount`.

Reply:
Yep, I use the full word "Maximum" for my control in CardsView plugin, but i followed spec

Reply:
Unless @rmarinho has an objection, I think we should use `Maximum`.

Reply:
@hartez @adrianknight89  well, let's go ahead with Maximum